app/services/bulk_zombie_urls/file_validator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class BulkZombieUrls::FileValidator
+  MAXIMUM_FILE_SIZE = 4.megabytes
```

Comment:
Where this this restriction comes from? I don't see it as part of the ticket.

Reply:
It's not the requirement from the ticket, but I am following this as per the previous other upload processes.